packages/adp-tooling/src/types.ts

```diff
@@ -199,14 +199,22 @@ export interface CodeExtChange extends CommonChangeProperties {
     };
 }
 
-export const CUSTOMER_BASE = 'CUSTOMER_BASE';
-
 export const enum TemplateFileName {
     Fragment = 'fragment.xml',
     Controller = 'controller.ejs',
     Annotation = 'annotation.xml'
 }
 
+export const enum UserState {
+    customer = 'CUSTOMER_BASE',
+    vendor = 'VENDOR'
+}
```

Comment:
I do not like the naming here: this is someting that we already call `UI5FlexLayer`. We have such type.

Also, later we have comparison like:
`layer === UserState.customer`

I does not look right to compare layer with user.

Reply:
Renamed with: 64b643d2688056bfb5325a849673fa0b8bc7f6df